resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/security/HadoopFSCredentialProvider.scala

```diff
@@ -48,9 +50,16 @@ private[security] class HadoopFSCredentialProvider
     val tmpCreds = new Credentials()
     val tokenRenewer = getTokenRenewer(hadoopConf)
     hadoopFSsToAccess(hadoopConf, sparkConf).foreach { dst =>
-      val dstFs = dst.getFileSystem(hadoopConf)
-      logInfo("getting token for: " + dst)
-      dstFs.addDelegationTokens(tokenRenewer, tmpCreds)
+      try {
+        val dstFs = dst.getFileSystem(hadoopConf)
+        logInfo("getting token for: " + dst)
+        dstFs.addDelegationTokens(tokenRenewer, tmpCreds)
+      } catch {
+        case e: StandbyException =>
+          logWarning(s"Namenode ${dst} is in state standby", e)
```

Comment:
hum..Here is actually fetching tokens from hadoopFS, including in hadoopFSCredentialProvider, which means it's exactly HDFS?